code-oss.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+set -e
+shopt -s nullglob
+
+function msg() {
+  echo "flatpak-vscode-oss: $*" >&2
+}
+
+function exec_vscode() {
+  exec /app/main/bin/code-oss \
+       --extensions-dir="${XDG_DATA_HOME}/vscode/extensions" \
+       "$@"
+}
+
+if [ -n "${FLATPAK_VSCODE_ENV}" ]; then
+  msg "Environment is already set up"
+  exec_vscode "$@"
+fi
+
+PYTHON_SITEDIR=$(python3 <<EOFPYTHON
+import sysconfig
+print(sysconfig.get_path('purelib', scheme='posix_prefix', vars={'base': '.'}))
+EOFPYTHON
+)
+
+for tool_dir in /app/tools/*; do
+  tool_bindir="$tool_dir/bin"
+  if [ -d "$tool_bindir" ]; then
+    msg "Adding $tool_bindir to PATH"
+    export PATH=$PATH:$tool_bindir
+  fi
+  tool_pythondir="$tool_dir/$PYTHON_SITEDIR"
+  if [ -d "$tool_pythondir" ]; then
+    msg "Adding $tool_pythondir to PYTHONPATH"
+    if [ -z "$PYTHONPATH" ]; then
+      export PYTHONPATH="$tool_pythondir"
+    else
+      export PYTHONPATH="$PYTHONPATH:$tool_pythondir"
+    fi
+  fi
+done
+
+if [ "$FLATPAK_ENABLE_SDK_EXT" = "*" ]; then
```

Comment:
I'd change it to always load all available SDK extensions. That's more in line with users' expectations that Flatpak should magically pick up what has been installed.

Reply:
It's the same behavior as of proprietary VS Code. Enabling all extensions unconditionally (at least via `enable.sh`) may result in conflicts, e.g. `node10` gets first on `PATH`, but `npm_config_nodedir` points to `node12` when both are installed.

Reply:
Fair enough.